opensrs/xcp.py

```diff
@@ -1,10 +1,12 @@
 import hashlib
 import logging
-import urllib2
+try:
+    from urllib.request import urlopen, Request
+except ImportError:
+    from urllib2 import urlopen, Request
 from xml.etree import ElementTree as ET
 
-from errors import XCPError
-
+from .errors import XCPError
```

Comment:
Let's be explicit, and import that from `opensrs.errors`. We generally try to avoid relative imports.